tests/testthat/test-ensembleMethods.R

```diff
@@ -176,22 +179,26 @@ context("Residual extraction")
 test_that("Residuals provided by residuals are proper for ensemble objects", {
   skip_on_cran()
   set.seed(2239)
-  ens.class <- caretEnsemble(models.class, trControl=trainControl(method="none"))
+  ens.class <- caretEnsemble(models.class, trControl = trainControl(method = "none"))
   models.subset <- models.reg[2:4]
   class(models.subset) <- "caretList"
-  ens.reg <- caretEnsemble(models.subset, trControl=trainControl(method="none"))
-  residTest <- residuals(ens.class)
-  residTest2 <- residuals(ens.reg)
+  ens.reg <- caretEnsemble(models.subset, trControl = trainControl(method = "none"))
+  suppressWarnings({
+    residTest <- residuals(ens.class)
+    residTest2 <- residuals(ens.reg)
+  })
   obs1 <- ifelse(Y.class == "No", 0, 1)
   obs2 <- Y.reg
-  predTest <- predict(ens.class)
-  predTest2 <- predict(ens.reg)
-  #expect_identical(residTest, obs1 - predTest)
-  #expect_identical(residTest2, obs2 - predTest2)
-  expect_false(identical(residTest2, predTest2 -obs2))
-
-  mr1 <- multiResiduals(ens.class)
-  mr2 <- multiResiduals(ens.reg)
+  expect_warning(predTest <- predict(ens.class))
```

Comment:
Why does this yield a warning now?

Reply:
In this line, `predict` is called without the `newdata` argument, as before. I suppose that this line was displaying an error previously.